### ISSUE TITLE
 If you stop weedvolume on one of the servers (any) Stops recording to other servers

### ISSUE THREAD

**Issue by YevgeniyGapchenko** [seaweedfs/seaweedfs#4638]
If you stop weedvolume on one of the servers (any) Stops recording to other servers
Works only for reading. Why do other servers not work for recording if one of the three is turned off?
How to fix it!

Server -ip=192.168.0.52
weed master -defaultReplication=201 -mdir="/data/seaweedfs/master" -port=9335 -peers=192.168.0.56:9333,192.168.0.55:9334,192.168.0.52:9335 -ip=192.168.0.52
weed volume  -dataCenter=dc2 -rack=rack1 -dir="/data/pictures/vol1" -max=10 -mserver="192.168.0.56:9333,192.168.0.55:9334,192.168.0.52:9335" -port=8081
weed volume  -dataCenter=dc2 -rack=rack1 -dir="/data/pictures/vol2" -max=10 -mserver="192.168.0.56:9333,192.168.0.55:9334,192.168.0.52:9335" -port=8084
Server -ip=192.168.0.55
weed master -defaultReplication=200 -mdir="/data/seaweedfs/master" -port=9334 -peers=192.168.0.56:9333,192.168.0.55:9334,192.168.0.52:9335 -ip=192.168.0.55
weed volume -dataCenter=dc3 -rack=rack1  -dir="/data/pictures/vol1" -max=10 -mserver="192.168.0.56:9333,192.168.0.55:9334,192.168.0.52:9335" -port=8081
ExecStart=/usr/local/bin/weed volume -dataCenter=dc3 -rack=rack2 -dir="/data/pictures/vol2" -max=10 -mserver=""192.168.0.56:9333,192.168.0.55:9334,192.168.0.52:9335"" -port=8080
Server -ip=192.168.0.56
weed master -defaultReplication=200 -mdir="/data/seaweedfs/master" -port=9333 -peers=192.168.0.56:9333,192.168.0.55:9334,192.168.0.52:9335 -ip=192.168.0.56
weed volume -dataCenter=dc1 -rack=rack2 -dir="/data/pictures/vol2" -max=5  -mserver="192.168.0.56:9333,192.168.0.55:9334,192.168.0.52:9335" -port=8080
weed volume -dataCenter=dc1 -rack=rack1 -dir="/data/pictures/vol1" -max=10 -mserver="192.168.0.56:9333,192.168.0.55:9334,192.168.0.52:9335" -port=8081

os version
Description:    Ubuntu 18.04.1 LTS
Release:        18.04
Codename:       bionic

weed version
version 30GB 3.53  linux amd64

I also want to pay attention to the server -ip=192.168.0.52
I have -defaultReplication=201 vol1 and vol2. If I try to synchronize them with the command <weed shell volume.balance -force> the figure below is not clear what it does, there is no result!
How to start synchronization correctly
volumes.
I would appreciate detailed instructions.
It is also not clear whether I need "weed filer"

![image](https://github.com/seaweedfs/seaweedfs/assets/86773660/46b85fcb-a83e-4647-b32d-5d8e2430e59a)
![image](https://github.com/seaweedfs/seaweedfs/assets/86773660/94d672de-f5ab-47ed-b59d-e107506fc558)
![image](https://github.com/seaweedfs/seaweedfs/assets/86773660/ce30f36f-f236-4393-a200-808cfdd05388)

And one more feature. Files can be written only by referring to the master-leader. How can I indicate in time which server to contact for recording if the leader has changed?

**Comment by kmlebedev** [comment]
Are you sure you need default Replication=201 ?
Did you read the [url](https://github.com/seaweedfs/seaweedfs/wiki/Replication#the-meaning-of-replication-type) carefully ?
Do you have exactly enough volumes for 3 replicas, taking into account the stop of one?

**Comment by YevgeniyGapchenko** [comment]
You can set it to 200. But nevertheless, the question remains, because the "weed shell volume.fix.replication" command also draws dots and does nothing! How to synchronize between servers, if necessary? Also, other questions remain open.
 I need to set up three masters so that the data between them is constantly synchronized, and it is desirable to contact any server for writing, and not just the leader.
I'm researching seaweedfs and seeing how I can apply it myself. And while it is bad understanding what and for what to use. We need to store pictures, a lot! And that they are always available.

**Comment by kmlebedev** [comment]
If it's simple then you need to build a failover cluster with all its components
1. master in the amount of 3 or more instances
2. external database in the amount of 3 3 or more instances
3. filer in the amount of 2 or more on the DC
4. volume in number of replica factor + 2
more detail see https://github.com/seaweedfs/seaweedfs/wiki/Production-Setup

Regarding draws dots, it seems you can set up verbose mode through `-v 9`

**Comment by YevgeniyGapchenko** [comment]
I read the instructions several times, but it's not entirely clear. We need more precise examples, a clear sequence of points. In the documentation I see only a vague description for each module. But it is difficult to assemble the system as a whole according to the documentation! For a whole week, through trial and error, I collect a stack, but I don’t get the result. Thanks for answers. I'll try to ask questions in the community.

**Comment by kmlebedev** [comment]
Yes, the complexity is due to the fact that this is a distributed file system and, accordingly, you need to have experience in building distributed and fault-tolerant systems at the scale you need.

There are examples of docker files on how to deploy a minimal cluster
https://github.com/seaweedfs/seaweedfs/blob/master/docker/compose/local-clusters-compose.yml
